java/src/jmri/jmrix/openlcb/swing/stleditor/StlEditorPane.java

```diff
@@ -749,23 +770,53 @@ private void decode(GroupRow groupRow) {
                             name = token;
                         }
                     }
+                    continue;
                 }
+                log.warn("STL Token \"{}\" ignored in \"{}\"", token, lines[i]);
             }
 
             var logic = new LogicRow(label, oper, name, comment);
             groupRow.getLogicList().add(logic);
         }
     }
-
-    private Operator getEnum(String name) {
+    
+    
+    /**
+     * Handle an operator and operand without
+     * an intervening space.
+     * @param name operand or concatenated operand and argument
+     * @return index between operator and operand
+     *          or -1
+     */
+    static int findOperatorBoundary(String name) {
+        // Works by finding, from the back, the 
+        // longest valid operator string.
+        // Efficient enough in the most common case of 
+        // just an operator.
+        //
+        // This implementation will parse "SET0" as a 
+        // "SET" command with an argument of "0", even though
+        // it might be intended to be an "SE" with an argument of "T0"".
+        // ("SET" takes no argument, which is how you tell these apart)
```

Comment:
I might misunderstand this comment, but I find it really confusing. On one hand, you write that `SET0` will be parsed as `SET` with parameter `0` and on the other hand, you write that `SET` cannot have any parameters.

I think I understand what you actually mean, but I find the comment very confusing.

Reply:
Thanks for the suggestions.  Have tried to improve both.